contrib/win32/win32compat/w32fd.c

```diff
@@ -1071,7 +1071,9 @@ spawn_child_internal(const char* cmd, char *const argv[], HANDLE in, HANDLE out,
 	if (strstr(cmd, "sshd.exe")) {
 		flags |= DETACHED_PROCESS;
 	}
-
+	if (strstr(cmd, "helper.exe")) {
```

Comment:
Please be explicit with the binary names..

change this to 
ssh-pkcs11-helper.exe, ssh-sk-helper.exe

Reply:
Changed.